src/app/content/components/Sidebar.tsx

```diff
@@ -87,6 +95,8 @@ const SidebarControl = styled(({isOpen, ...props}: React.HTMLProps<HTMLButtonEle
 
 `;
 
+const SidebarControl = injectIntl(StyledSidebarControl);
```

Comment:
is this `injectIntl` necessary to programmatically access the messages with `intl.formatMessage` rather than the `FormattedMessage` component?

Reply:
Yes, I used it in order to set attributes.

Reply:
if you use the `FormattedMessage` with the render function you could accomplish the same thing. i suppose its useful to follow both patterns though because if you had multiple messages on different attributes nested `FormattedMessage` components would be super gross